app/decorators/talk_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class TalkDecorator < Draper::Decorator
+  delegate_all
+
+  def date
+    model.date.to_date.to_fs(:date)
```

Comment:
the `#to_date` should no be necessary here

Reply:
It is necessary when it's a datetime.

Reply:
have you tried ?

Reply:
I have.
```ruby
> Talk.last.date.to_date.to_fs(:date)
=> "08/05/2023"
> Talk.last.date.to_fs(:date)
=> "2023-05-08 12:07:14 -0300"
```

Reply:
🤔 the strftime should get the date part only
anyway, 👍🏻